README.md

```diff
@@ -1,22 +1,43 @@
 # openSUSE News
-openSUSE News Website, providing news to the community
+
+openSUSE News Website, providing news to the community.
+
+## How to submit a new post?
+
+1. Create a new markdown file in `_posts` directory. File name should follow
+    `yyyy-mm-dd-title.md` format. For example, `2020-02-27-my-first-post.md`.
+    You can start by clicking [this link](https://github.com/openSUSE/news-o-o/new/master/?filename=_posts/2020-02-27-my-first-post.md).
+2. Write content. You can start with the following template:
+    ```
+    ---
+    author: Stasiek Michalski
+    date: 2020-02-27 19:27:17+00:00
+    layout: post
+    title: My First Post
+    image: /assets/images/omw.png
+    categories: Announcements
+    tags: infrastructure websites
```

Comment:
Tags and Categories need to be in the following formats:
`[tag1, tag2]`
or
```
- tag1
- tag2
```

Reply:
fixed